files/usr/lib/jeos-firstboot

```diff
@@ -26,6 +26,9 @@ TEXTDOMAIN='jeos-firstboot'
 . /etc/os-release
 . "$0-functions"
 
+# Read the optional configuration file
+[ -f /etc/jeos-firstboot.conf ] && . /etc/jeos-firstboot.conf
```

Comment:
It should either do

```
[ -f /usr/share/defaults/jeos-firstboot.conf ] && . /usr/share/defaults/jeos-firstboot.conf
[ -f /etc/jeos-firstboot.conf ] && . /etc/jeos-firstboot.conf
```

or 

```
if [ -f /etc/jeos-firstboot.conf ]; then
    . /etc/jeos-firstboot.conf
elif [ -f /usr/share/defaults/jeos-firstboot.conf ]; then
    . /usr/share/defaults/jeos-firstboot.conf
fi
```

Reply:
Right. I changed the path of the initial example too.

Done.